tests/core/utilities/test_abi.py

```diff
@@ -510,14 +535,6 @@ def test_get_abi_element(
             TypeError,
             "Unsupported function identifier",
         ),
-        (
-            CONTRACT_ABI,
```

Comment:
Meant to move this to the `test_get_abi_element`, since now `get_abi_element` will determine the function by matching it's name (without arguments). Since `logTwoEvents` is the only function with that name, this case works.

I'll add a new failing case for an ambiguous function name that raises a MismatchedABI.